pkgs/tools/security/onlykey-cli/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, python3Packages, stdenv }:
```

```suggestion
{ lib, python3Packages }:
```

stdenv is not actually used.